data-generation/src/test/scala/LandySpec.scala

```diff
@@ -59,4 +60,17 @@ class LandySpec extends AnyFlatSpec with SparkTestWrapper {
 
     assert(intervalWithAll._1.count() == 2 && intervalWithV1andEdge._2.count() == 1)
   }
+
+  it can "query direct neighbours" in {
+    implicit val sparkContext: SparkContext = spark.sparkContext
+    val g = new Landy(createGraph())
```

Comment:
createGraph bare lager en graph? Som du vet i hodet ditt hvordan ser ut liksom?

Reply:
Ye, litt lettere å gjenbruke, men er jo litt dumt at man har masse magiske verdier lenger ned i test casen. Er så sykt tedious å generere logger for hver greie og lime dem sammen.

Reply:
Ye, litt lettere å gjenbruke, men er jo litt dumt at man har masse magiske verdier lenger ned i test casen. Er så sykt tedious å generere logger for hver greie og lime dem sammen.